### DISCUSSION TITLE
Keyboard navigation in DateInput

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#5542] [Q&A]
I like the Mantine DateInput component. 😊

Can I have keyboard navigation in the calendar component like in DatePickerInput?
I know it's not working because the input component catches the keydown event. Is there a way to have both features? 🤔 

Thanks! 🙏

**Comment by ryuhangyeong** [comment]
When I tested it, you can navigate with Tab and Arrow Keys!
